backend/src/db/migrations/20251127192155_adds-scope-org-id-to-identity-access-tokens.ts

```diff
@@ -0,0 +1,22 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
```

Comment:
Ignore comment placement. 

When i switch from a suborganization project to organization using breadcrumbs 

<img width="1060" height="149" alt="Image" src="https://github.com/user-attachments/assets/0ff049d9-08f2-452f-bc67-e4cd6d7029d8" />